gabbi/exception.py

```diff
@@ -0,0 +1,18 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+"""Gabbi specific exceptions."""
```

Comment:
Yeah, I'm undecided about that, mostly because of the way I've been doing the "from package import module" form of methods. In which case when you want to use the GabbiFormatError you use it as `exception.GabbiFormatError` or `exceptions.GabbiFormatError` and the former sounds nicer somehow.

I don't really care one way or another though. The main goal here was to put it somewhere that could be shared from multiple places.